encryption/aes/aes.go

```diff
@@ -4,23 +4,25 @@ import (
 	"crypto/aes"
 	"crypto/cipher"
 	"crypto/rand"
+	"encoding/base64"
 	"encoding/hex"
 	"fmt"
 	"io"
+
+	"github.com/wego/pkg/encryption"
+	"github.com/wego/pkg/errors"
 )
 
 // KeyLength is the min length of the secret key
 const KeyLength = 32
 
-// errors
+// error
 var (
-	ErrShortKey         = fmt.Errorf("key is too short, %d bytes is required", KeyLength)
-	ErrInvalidHexString = fmt.Errorf("ciphertext is not a valid hex string")
-	ErrShortData        = fmt.Errorf("data is too short")
+	ErrShortKey = fmt.Errorf("key is too short, %d bytes is required", KeyLength)
```

Comment:
is it 32 bytes or 32 characters?

Reply:
It's bytes.
But when passing as a string, I'm checking for 32 characters.
So if the caller uses a UTF-8 string as a key, it might cause confusion.
Any suggestion?

Reply:
oh ok. I think it shld be fine. the caller should be aware if it is using a UTF-8 string